back/locale/en/LC_MESSAGES/messages.po

```diff
@@ -1,34 +1,47 @@
 msgid ""
-msgstr ""
-"Content-Type: text/plain; charset=UTF-8\n"
+msgstr "Content-Type: text/plain; charset=UTF-8\n"
```

Comment:
I think this is wrong, empty msgid and msgstr just mark the start of headers, which are just strings